adoc/admin-monitoring-horizontal-pod-autoscaler.adoc

```diff
@@ -0,0 +1,293 @@
+- *AverageValue*: the value returned from the metrics server API is divided by the number of all relevant pods, then compared to the `metrics.target.averageValue`.
+- *Value*: the value returned from the metrics server API is directly compared to the `metrics.target.value`.
+
+[NOTE]
+====
+The metrics supported by `metrics-server` are the *CPU* and *memory of a pod or node*.
+====
+
+[IMPORTANT]
+====
+API versions supported by the HPA:
+
+- CPU metric: `autoscaling/v1`,`autoscaling/v2beta2`
+- Memory metric: `autoscaling/v2beta2`.
+====
+
+== Usage
+
+It is useful to first find out about the available resources of your cluster.
+
+* To display resource (CPU/Memory) usage for nodes, run:
++
+[source,bash]
+----
+$ kubectl -n kube-system top node
+----
++
+the expected output should look like the following:
++
+[source,bash]
+----
+NAME        CPU(cores)   CPU%   MEMORY(bytes)   MEMORY%
+master000   207m         10%    1756Mi          45%
+worker000   100m         10%    602Mi           31%
+----
+
+* To display resource (CPU/Memory) usage for nodes, run:
++
+[source,bash]
+----
+$ kubectl -n kube-system top pod
+----
++
+the expected output should look like the following:
++
+[source,bash]
+----
+NAME                                CPU(cores)   MEMORY(bytes)
+cilium-9fjw2                        32m          216Mi
+cilium-cqnq5                        43m          227Mi
+cilium-operator-7d6ddddbf5-2jwgr    1m           46Mi
+coredns-69c4947958-2br4b            2m           11Mi
+coredns-69c4947958-kb6dq            3m           11Mi
+etcd-master000                      21m          584Mi
+kube-apiserver-master000            20m          325Mi
+kube-controller-manager-master000   6m           105Mi
+kube-proxy-x2965                    0m           24Mi
+kube-proxy-x9zlv                    0m           19Mi
+kube-scheduler-master000            2m           46Mi
+kured-45rc2                         1m           25Mi
+kured-cptk4                         0m           25Mi
+metrics-server-79b8658cd7-gjvhs     1m           21Mi
+oidc-dex-55fc689dc-f6cfg            1m           20Mi
+oidc-gangway-7b7fbbdbdf-85p6t       1m           18Mi
+----
++
+[NOTE]
+====
+The option flag `--sort-by=cpu`/`--sort-by=memory` has an sorting issue at the moment. It will be fixed in the future.
+====
+
+=== Using Horizontal Pod Autoscaler (HPA)
+
+You can set the HPA to scale according to various metrics.
+These include *average CPU utilization*, *average CPU value*, *average memory utilization* and *average memory value*. The following sections show the recommended configuration for each of the aforementioned options.
+
+==== Creating an HPA Using Average CPU Utilization
+
+The following code is an example of what this type of HPA can look like:
+
+TODO::
+  Where does this configuration go? (which config file?)
```

Comment:
@jenting ah, sorry, I marked the question as TODO. I am not sure where the following configuration goes.